cli-scripts/get-credentials.js

```diff
@@ -0,0 +1,113 @@
+const fs = require("fs/promises");
+const path = require("path");
+const util = require("util");
+const exec = util.promisify(require("child_process").exec);
```

Comment:
Since it looks like you are awaiting allot of these functions anyways you can use the synchronous version of these and remove all of the awaits and asyncs it will make the code more readable.

https://nodejs.org/api/child_process.html#child_processexecsynccommand-options
https://nodejs.org/api/fs.html#fs_fs_writefilesync_file_data_options

Reply:
I've switched to the synchronous versions of those.

It didn't look to me like there was a synchronous version of the AWS calls so I left those be. 
Please bear with me if I'm wrong about that (I'm new at this 😄)